src/adapter/src/coord.rs

```diff
@@ -2532,11 +2540,53 @@ impl Coordinator {
             .await
             .unwrap_or_terminate("cannot fail to create collections");
 
+        self.bootstrap_builtin_continual_tasks(new_builtin_continual_tasks)
+            .await;
+
         if !self.controller.read_only() {
             self.apply_local_write(register_ts).await;
         }
     }
 
+    /// TODO(ct1): Hack to make as_of selection happy. It assumes that anything
+    /// with a zero upper has the since set to the initial as_of. Ideally we'd
```

Comment:
This says "anything with a zero upper has the since set to the initial as of" but it seems like this is really invoked for every builtin CT on every boot? Might still be fine (or I might be misreading)—just wanted to call out that I couldn't tie this comment to what was actually happening in the code!

Reply:
Ah ack. I was thinking of this when I wrote the comment, but sounds like I didn't quite manage to nail it. Will take another pass (at some point, don't want to interrupt CI)

Reply:
This got lost at the bottom of my TODO list for a while, but I did finally get back around to it! https://github.com/MaterializeInc/materialize/pull/30228